src/data/pnat/basic.lean

```diff
@@ -349,4 +350,122 @@ theorem gcd_mul_lcm (n m : ℕ+) : (gcd n m) * (lcm n m) = n * m :=
 
 def prime (p : ℕ+) : Prop := (p : ℕ).prime
 
+/-- Two pnats are coprime if their gcd is 1. -/
+def coprime (m n : ℕ+) : Prop := m.gcd n = 1
+
+@[simp]
+lemma eq_iff {m n : ℕ+} :
+(m : ℕ) = ↑n ↔ m = n := by { split, apply eq, intro h, rw h }
+
+@[simp]
+lemma coprime_coe {m n : ℕ+} : nat.coprime ↑m ↑n ↔ m.coprime n :=
+by { unfold coprime, unfold nat.coprime, rw ← eq_iff, simp }
+
+lemma coprime.mul {k m n : ℕ+} : m.coprime k → n.coprime k → (m * n).coprime k :=
```

Comment:
```suggestion
lemma coprime.mul_left {k m n : ℕ+} : m.coprime k → n.coprime k → (m * n).coprime k :=
```
Please also add `mul_right`.
Also, these are iff statements, right? So you could write
```lean
(m *n).coprime k \iff (m.coprime k \and n.coprime k) := _
```
I guess that other direction will occasionally be useful as well.
```

Reply:
This is awkward... the corresponding lemmas are called nat.coprime.mul and nat.coprime.mul_right. I'd change the name on nat.coprime.mul also, but I don't know how to find and replace it across everything.

Reply:
Ooh, then don't bother. Just do whatever is done for `nat.coprime`.